system/jlib/jarray.hpp

```diff
@@ -67,9 +66,9 @@ class jlib_decl Allocator
     void doSwapWith(Allocator & other);
 
 protected:
-    void * _head;
-    aindex_t used;
-    aindex_t max;
+    void * _head = nullptr;
```

Comment:
This change was needed to work around a problem with the order that constructors for static objects were called.